Source/Scene/Cesium3DTileset.js

```diff
@@ -270,7 +274,8 @@ define([
             lastColor : new Cesium3DTilesetStatistics(),
             lastPick : new Cesium3DTilesetStatistics()
         };
-
+        
+        this._optimizations = defaultValue(options.optimizations, new Cesium3DTileOptimizations());
```

Comment:
Just remove this for now until we decide on https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/176